lib/roles_conversion.rb

```diff
@@ -0,0 +1,40 @@
+class RolesConversion
```

Comment:
I was trying to avoid the word "migration".

The class represents one-time transformation of data storage from environment variable to User-with-Role. I guess it could be `RolesTransformer` or `RolesWithButter` or something else innocuous. I imagine it might be used again when transforming GSA18F_PURCHASER_EMAIL and GSA18F_APPROVER_EMAIL in a similar way. I am not attached to what it is called.